subsys/emul/espi/espi_emul_host.c

```diff
@@ -0,0 +1,388 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Emulator for the generic eSPI Host. This supports basic
+ * host operations.
+ */
+
+#define DT_DRV_COMPAT zephyr_espi_emul_host
+
+#define LOG_LEVEL CONFIG_ESPI_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(espi_host);
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/emul.h>
+#include <zephyr/drivers/espi.h>
+#include <zephyr/drivers/espi/espi_utils.h>
+#include <zephyr/drivers/espi_emul.h>
+
+/** Declare the default state of virtual wires */
+const static struct espi_emul_vw_data vw_state_default[] = {
+	{ ESPI_VWIRE_SIGNAL_SLP_S3,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SLP_S4,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SLP_S5,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_OOB_RST_WARN,	0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_PLTRST,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SUS_STAT,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_NMIOUT,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SMIOUT,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_HOST_RST_WARN,	0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SLP_A,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SUS_PWRDN_ACK,	0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SUS_WARN,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SLP_WLAN,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_SLP_LAN,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_HOST_C10,		0, ESPI_MASTER_TO_SLAVE },
+	{ ESPI_VWIRE_SIGNAL_DNX_WARN,		0, ESPI_MASTER_TO_SLAVE },
+};
```

Comment:
Same here as `espi_emul.c`, please use the dot notation for the specific fields

Reply:
Done